cisco_secure_endpoint/assets/dashboards/cisco_secure_endpoint_event.json

```diff
@@ -0,0 +1,1949 @@
+{
+  "title": "Cisco Secure Endpoint - Event",
+  "description": " This dashboard provides detailed insights into the event logs generated by Cisco Secure Endpoint.",
```

Comment:
```suggestion
  "description": "This dashboard provides detailed insights into the event logs generated by Cisco Secure Endpoint.",
```

Reply:
Thanks for the comment. Have made the changes.